packages/code-studio/src/styleguide/ListViews.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useCallback, useState } from 'react';
+import {
+  Grid,
+  Icon,
+  Item,
+  ListView,
+  ItemKey,
+  Text,
+} from '@deephaven/components';
+import { vsAccount, vsPerson } from '@deephaven/icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { generateNormalizedItems, sampleSectionIdAndClasses } from './utils';
+
+// Generate enough items to require scrolling
+const itemsSimple = [...generateNormalizedItems(52)];
+
+function AccountIllustration(): JSX.Element {
+  return (
+    // Images in ListView items require a slot of 'image' or 'illustration' to
+    // be set in order to be positioned correctly:
+    // https://github.com/adobe/react-spectrum/blob/784737effd44b9d5e2b1316e690da44555eafd7e/packages/%40react-spectrum/list/src/ListViewItem.tsx#L266-L267
```

Comment:
Any reason they don't have a slot provider for `icon`? We could even add a PR for it... seems odd they support `image` and `illustration` but not `icon`.

Reply:
I'm guessing @dsmmcken suggestion may be accurate here. Maybe they are trying to not overload the fact that icons can be included in actions.

Reply:
Yeah, that's a key difference between listview and listbox, https://react-spectrum.adobe.com/react-spectrum/ListBox.html#complex-items